<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
font-family:century gothic;
font-size:15px
}
/*QSlider::groove:horizontal {
                border: 1px solid #999999;
                height: 32px;
				background-color: white;
                 }
            QSlider::handle:horizontal {
                background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b4b4b4, stop:1 #8f8f8f);
                border: 1px solid #5c5c5c;
                width: 5px;
                margin: -20px 0;
                border-radius: 3px;}*/
QSlider::handle:horizontal{background-color:#979797}
            
ImageView{
border-radius:100px;}
QMainWindow{
background-color: #C7D3FF;
}
QComboBox{
background-color:white;
border-top:none;}
QMenu{
background-color:#BCD0FF;
}
QLabel{
font-size:18px}
QLabel#label{font-size:26px; text-decoration:underline;}
QLabel#label_2{font-size:26px; text-decoration:underline;}
QLabel#label_3{font-size:26px; text-decoration:underline;}
QLabel#label_4{font-size:26px;  text-decoration:underline;}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="1" column="1">
     <widget class="QWidget" name="gridWidget_4" native="true">
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="3" column="1">
        <widget class="ImageView" name="viewOutput2"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>OutPut 1</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="ImageView" name="viewOutput1"/>
       </item>
       <item row="1" column="1" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>OutPut 2</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="Line" name="line_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="gridWidget" native="true">
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="3" column="0">
        <widget class="ImageView" name="viewFirstimg">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="ImageView" name="viewFirstComponent"/>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select A Component</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Imaginary</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>First Image</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="Line" name="line_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QWidget" name="gridWidget_2" native="true">
      <property name="maximumSize">
       <size>
        <width>1000</width>
        <height>400</height>
       </size>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_4">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select Image</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Image1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Image2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboBox_5">
         <property name="minimumSize">
          <size>
           <width>100</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select Image</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Image1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Image2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Component 2</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_6">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select Component</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Imaginary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UniMagnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UniPhase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select Output Window</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Output 1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Output 2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Mixer Output to:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1" colspan="2">
        <widget class="QComboBox" name="comboBox_7">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select A Component</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Imaginary</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UniMagnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>UniPhase</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QSlider" name="horizontalSlider_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Component 1</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QSlider" name="horizontalSlider">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QWidget" name="gridWidget_3" native="true">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="3" column="0">
        <widget class="ImageView" name="viewSecondimg"/>
       </item>
       <item row="3" column="1">
        <widget class="ImageView" name="viewSecondComponent"/>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox2">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>25</height>
          </size>
         </property>
         <item>
          <property name="text">
           <string>Select A Component</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Magnitude</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Phase</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Real</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Imaginary</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="0" alignment="Qt::AlignHCenter">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Second Image</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="Line" name="line_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuAdd">
    <property name="title">
     <string>Add</string>
    </property>
    <addaction name="openFirstImg"/>
    <addaction name="openSecondImg"/>
   </widget>
   <addaction name="menuAdd"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openFirstImg">
   <property name="text">
    <string>Add First Image</string>
   </property>
  </action>
  <action name="openSecondImg">
   <property name="text">
    <string>Add Second Image</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QGraphicsView</extends>
   <header>pyqtgraph</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>comboBox</tabstop>
  <tabstop>comboBox2</tabstop>
  <tabstop>comboBox_3</tabstop>
  <tabstop>comboBox_4</tabstop>
  <tabstop>comboBox_6</tabstop>
  <tabstop>horizontalSlider</tabstop>
  <tabstop>comboBox_5</tabstop>
  <tabstop>comboBox_7</tabstop>
  <tabstop>horizontalSlider_2</tabstop>
  <tabstop>viewFirstimg</tabstop>
  <tabstop>viewFirstComponent</tabstop>
  <tabstop>viewSecondimg</tabstop>
  <tabstop>viewOutput1</tabstop>
  <tabstop>viewSecondComponent</tabstop>
  <tabstop>viewOutput2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
